src/openklant/components/klantinteracties/api/tests/test_expand.py

```
@@ -382,3 +382,83 @@ def test_detail_multiple_level_expansion(self):
                 "land": self.partij.correspondentieadres_land,
             },
         )
+
+    def test_invalid_multiple_params_expansion(self):
+        detail_url = reverse(
+            "klantinteracties:klantcontact-detail",
+            kwargs={"uuid": str(self.klantcontact.uuid)},
+        )
+
+        response = self.client.get(detail_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue("_expand" not in response.json())
+
+        explicit_url = detail_url + "?expand="
+        response = self.client.get(explicit_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue("_expand" not in response.json())
+
+        explicit_url = detail_url + "?wrong_expand="
+        response = self.client.get(explicit_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue("_expand" not in response.json())
+
+        explicit_url = detail_url + "?expand[]="
+        response = self.client.get(explicit_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue("_expand" not in response.json())
+
+        explicit_url = detail_url + "?expand[]=hadBetrokkenen"
+        response = self.client.get(explicit_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue("_expand" not in response.json())
+
+        explicit_url = (
+            detail_url + "?expand[]=leiddeTotInterneTaken&expand[]=hadBetrokkenen"
+        )
+        response = self.client.get(explicit_url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

I'd prefer to raise a 400 error here, otherwise it might be unclear that this method of providing query params is not supported

---

That's my main comment too: don't mind not supporting `expand[]`, but my concern is that people will get a `200` and think all is fine, even though they're not actually passing the parameters they think they are.